lib/fedex/tracking_information.rb

```diff
@@ -47,11 +47,15 @@ def initialize(details = {})
       if details.has_key?(:estimated_delivery_timestamp)
         @estimated_delivery_at = Time.parse(details[:estimated_delivery_timestamp])
       end
+      
+      if details[:events].select { |event| event[:event_type] == "PU" }.first
```

Comment:
a) this feature sounds useful! so :+1:

b) I think this code can be slightly more compact:

```ruby
pickup_event = details[:events].detect { |event| event[:event_type] == "PU" }
if pickup_event
  @shipped_at = pickup_event[:timestamp]
end
```

c) Should we try to parse the timestamp?  Or does it come out of the hash as a legit Timestamp?